assets/vhosts/toran-proxy-http-reverse.conf

```diff
@@ -0,0 +1,21 @@
+server {
```

Comment:
@cedvan Well yes, but i guess there are probably more settings to come that we would need to adjust in a different manner from default http. But i will update that anyway. =)